lib/optimize/RemoveParentModulesPlugin.js

```diff
@@ -3,7 +3,11 @@
 	Author Tobias Koppers @sokra
 */
 function hasModule(chunk, module, checkedChunks) {
-	if(chunk.modules.indexOf(module) >= 0) return [chunk];
+	if(module.debugId && chunk.modulesIndex) {
+		if(module.debugId in chunk.modulesIndex) return [chunk];
```

Comment:
I've seen perf go down when using `in` operator.
`if(chunk.modulesIndex[module.debugId])` performs better.